### PR TITLE
add secondary (gray) default color

### PR DESCRIPTION
**Pull request by samanpwbb**
cc: @mayagao who will finish this PR up.

closes https://github.com/mapbox/assembly/issues/554

### CODE REVIEW

**Comment by mayagao** [comment]
also modified the blue to be less sharp

**Review by mayagao — APPROVED** [review]

**Comment by mayagao** [comment]
screen shot 2017-01-05 at 4 34 28 pm

change the color to blue in the last commit and made some style changes

File: src/forms.css [hunk 1]
```diff
@@ -444,14 +444,14 @@ because it seems to be more cross-browser consistent with all HTML5 input types
...
   cursor: pointer;
 }
 
 /* range thumb:hover */
-.range:hover::-webkit-slider-thumb { border-color: var(--default-interactive-color-dark); }
-.range:hover::-ms-thumb { border-color: var(--default-interactive-color-dark); }
-.range:hover::-moz-range-thumb { border-color: var(--default-interactive-color-dark); }
+.range:hover::-webkit-slider-thumb { border-color: var(--gray); }
+.range:hover::-ms-thumb { border-color: var(--gray); }
+.range:hover::-moz-range-thumb { border-color: var(--gray); }
```
Comment (samanpwbb) [inline src/forms.css:454]: It's better to use `default-` colors for defaults. You sure we  can't use secondary-dark?

File: src/variables.json [hunk 2]
```diff
@@ -65,16 +65,19 @@
...
+  "default-secondary-interactive-color": "#999",
 
   "border-radius": "4px",
   "border-radius-bold": "8px",
   "border-radius-full": "9999px",
 
   "transition": "0.125s",
 
-  "focus-shadow": "0 0 0 3px rgba(137, 199, 216, 0.65)",
+  "focus-shadow": "0 0 0 3px rgba(0, 0, 0, 0.35)",
```
Comment (samanpwbb) [inline src/variables.json]: Can you revert this change, or submit it as a separate PR?

File: site/home.js [hunk 3]
```diff
@@ -13,9 +12,8 @@ class Home extends React.Component {
     return (
       <div className='mt24 wmax960'>
         <div className='txt-headline'>
-          <h1 className='pl6 pr6 flex-parent mt72 flex-parent--center-cross'>
-            <Logo className='w48 inline-block mr12' style={{ fill: '#9762cc' }}/>
```
Comment (samanpwbb) [inline site/home.js:17]: We should show the logo on the home page!

**Comment by samanpwbb** [comment]
@mayagao looks like you need to update jest tests because default colors changed. You can do that with `npm run test:jest -- -u`

File: src/forms.css (hunk 1 above)
Comment (mayagao) [reply src/forms.css:454]: So I was thinking:

- for textboxes, blue on hover to  show more contrast 
- checkbox, radio button, and switches, blue on hover to show more contrast between different select states.
- for select and range input, dark gray on hover, blue seems to be too much
- not sure about toggles, could be blue or dark gray ...

File: site/home.js (hunk 3 above)
Comment (mayagao) [reply site/home.js:17]: it's already on the sidebar though ...

File: src/variables.json (hunk 2 above)
Comment (mayagao) [reply src/variables.json]: this was an accident reverted

**Comment by mayagao** [comment]
test updated

File: site/home.js (hunk 3 above)
Comment (samanpwbb) [reply site/home.js:17]: 🆗

File: src/forms.css (hunk 1 above)
Comment (samanpwbb) [reply src/forms.css:454]: Lets merge this and feel it out in some real projects – we can come back to it later.

**Comment by samanpwbb** [comment]
Nice work!
